libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -33,10 +36,13 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    // Using async pipe operator in html instead of subscribing in ts file
-    // No need to unsubscribe manually since we are using async in html
-    // Added change detection to onpush for faster page load
-    this.books$ = this.store.select(getAllBooks);
+    this.store.select(getAllBooks).subscribe(books => {
```

Comment:
This is taken care at line no 21 therefore not required here.

Reply:
Updated code.